app/services/vendor_validator_service.rb

```diff
@@ -0,0 +1,45 @@
+class VendorValidatorService
```

Comment:
Sure, I'll go with `SubmitIdvJob` and turn the other into `VendorValidatorResultStorage`